Workbooks/Network Insights/AzureFirewallWorkbooks/StructuredLogsWorkbook/StructuredLogs.workbook

```diff
@@ -0,0 +1,57 @@
+{
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "workbookDisplayName": {
+        "type": "string",
+        "defaultValue": "Azure Firewall Workbook",
+        "metadata": {
+          "description": "The friendly name for the workbook that is used in the Gallery or Saved List.  This name must be unique within a resource group."
+        }
+      },
+      "workbookType": {
+        "type": "string",
+        "defaultValue": "workbook",
+        "metadata": {
```

Comment:
you keep on using the ARM Template version of this file.
in this repo, it should just be the workbook content.
when exporting from the workbook editor, do NOT switch it to ARM template mode, leave it in "gallery template" mode.
@ericajlin  i'm surprised this .workbook file being an arm template doesn't fail template validation or loc validation, etc

Reply:
Updated